fairroot.sh

```diff
@@ -70,6 +70,13 @@ cmake $SOURCEDIR
 
 cmake --build . -- -j$JOBS install
 
+INCLUDE_TREE="$INSTALLROOT/include_tree"
+mkdir -p $INCLUDE_TREE
+SUBDIRS="source sink field event sim steer"
+for dir in $SUBDIRS; do
```

Comment:
Should be:

```bash
for DIR in source sing field event sim steer; do
  ln -nfs ../include $INSTALLROOT/include/$DIR
done
```

Reply:
Yep, I'm fixing this.

Please confirm if you still want me to use the single `include` dir for all the symlinks.

Reply:
Sure, I prefer to have a single `include` directory, thanks!